pkgs/native_synchronization/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: native_synchronization
+description: Low level synchronization primitives built on dart:ffi.
+version: 0.3.0
```

Comment:
Rev minor version for publish, pointing to the new repo.

Reply:
sgtm; will plan to do this as part of a follow-up PR